### PR TITLE
 Added Support for 64bit RGB format

### PR DESCRIPTION
**Pull request by pyn1**
Jira: https://jira01.devtools.intel.com/browse/OAM-52530
Tests: Test Passes
Signed-off-by: Poornima <poornima.y.n@intel.com>

### CODE REVIEW

**Comment by pyn1** [comment]
@kalyankondapally Please review.

File: os/android/utils_android.h [hunk 1]
```diff
@@ -58,6 +58,8 @@ static uint32_t GetDrmFormatFromHALFormat(int format) {
       return DRM_FORMAT_ARGB8888;
     case HAL_PIXEL_FORMAT_YV12:
       return DRM_FORMAT_YVU420;
+    case HAL_PIXEL_FORMAT_RGBA_FP16:
```
Comment (kalyankondapally) [inline os/android/utils_android.h]: Can you remove the OAM jira and in tests add what test starts passing with this change ?

Comment (pyn1) [reply os/android/utils_android.h]: Okay.

**Comment by pyn1** [comment]
Updated taking care of review comments. Please review.

**Pull request closed without merging**